fonzie/views/status.py

```diff
@@ -4,10 +4,9 @@
 """
 from __future__ import absolute_import, unicode_literals
 
-from rest_framework.response import Response
-from rest_framework.views import APIView
-
 from fonzie import __version__ as fonzie_version
+from rest_framework.response import Response  # pylint: disable=import-error
+from rest_framework.views import APIView  # pylint: disable=import-error
```

Comment:
Have you tried declaring rest_framework as THIRD_PARTY package in isort configuration?